### PR TITLE
Consumer key redirection support (fix #8)

### LINKED ISSUE #8: Missing consumer key redirection URL
The 'redirection' field in the 'CkRequest' struct used as body for the endpoint '/auth/credential' is missing.

### PR DESCRIPTION
**Pull request by thbkrkr**
Signed-off-by: thb thb.krkr@gmail.com

### CODE REVIEW

File: ovh/consumer_key.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ type CkValidationState struct {
 type CkRequest struct {
 	client      *Client
 	AccessRules []AccessRule `json:"accessRules"`
+	Redirection string       `json:"redirection"`
```
Comment (yadutaf) [inline ovh/consumer_key.go]: Can you make this parameter optional ? https://api.ovh.com/1.0/auth.json

**Comment by yadutaf** [comment]
Thanks for the fix! Can you add/fix related unit tests?

**Comment by thbkrkr** [comment]
I have added 'omitempty' to make the parameter optional.
It fixes the unit tests. I don't know what pertinent unit test to add.
I have also fixed a typo in the test function names.

**Comment by yadutaf** [comment]
Thanks ! I'd add a test to make sure ``NewCkRequestWithRedirection`` and then issuing a requests actually sends the redirection field

**Comment by yadutaf** [comment]
Merged. Thanks !

**Pull request merged**